apis/python/src/tiledb/vector_search/index.py

```diff
@@ -170,3 +179,140 @@ def query(
             )
 
         return np.array(r)
+
+    def distributed_query(
```

Comment:
How about we add a `mode` parameter to the existing `query` function? This way choosing between local and distributed execution will be more seamless.

Reply:
Done